### PR TITLE
Bug fix/search bar empty

### PR DESCRIPTION
**Pull request by stephanie-roe**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 
- Restructured NavBar component and logic to eliminate bug affecting live search. 

Were there any issues that arose? 
-Logic was flawed, but we got it working!! 

Is there anything you need from your teammates? 

Any other comments, questions, or concerns?

## Issues
Closes #

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [ ] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -40,86 +38,52 @@ export type Brewery = {
...
-              return (<Breweries newBrewery={this.state.searchedBreweries} />)
-            } else if (this.state.searchedBreweries && !this.state.error) {
+            if (!this.state.searchedBreweries.length && !this.state.query) {
               return (<Breweries newBrewery={this.state.breweries} />)
-            } else if (this.state.error) {
-              return (<h1>Sorry, come back later!</h1>)
+            } else if (!this.state.searchedBreweries.length) {
+              return (<h1>Oops, try again later!</h1>)
+            } else {
+              return (<Breweries newBrewery={this.state.searchedBreweries} />)
```
Comment (rachellhill) [inline src/Components/App.tsx:86]: Great work getting the logic to do what we need it to!

**Review by rachellhill — APPROVED** [review]
Awesome job working through this bug!

File: src/Components/NavBar.tsx [hunk 2]
```diff
@@ -1,67 +1,49 @@
-import React, { Component } from 'react';
+// import React, { Component } from 'react';
```
Comment (userigorgithub) [inline src/Components/NavBar.tsx:1]: Changing class component to a functional component, nice!

**Comment by aominhlong** [comment]
Good job working out the logic. Now we can add the gif to the background image and work on implementing other functionality for the App

File: src/Components/App.tsx [hunk 3]
```diff
@@ -40,86 +38,52 @@ export type Brewery = {
...
         console.log(response)
         return response.json()})
-      // .then(data => {
-      //   return data as Brewery[]
-      // })
   }
 
-  searchBrewery = (query: string, event: any): void => {
-    event.preventDefault()
+  searchBrewery = (event: any): void => {
```
Comment (userigorgithub) [inline src/Components/App.tsx:61]: GJ refactoring searchBrewery method.

**Review by userigorgithub — APPROVED** [review]
Amazing job not being afraid to do some major refactoring overall in 2 files!

**Review by aominhlong — APPROVED** [review]

**Pull request merged**